Resources/Prototypes/Entities/Objects/Devices/pda.yml

```diff
@@ -414,8 +414,8 @@
 - type: entity
   parent: BasePDA
   id: LawyerPDA
-  name: lawyer PDA
-  description: For lawyers to poach dubious clients.
+  name: attorney PDA
```

Comment:
not sure how necessary the lawyer rename is; as lawyer vs prosecution works regardless but if you're renaming upstream files this should be done via locale to avoid the forever eternal merge conflicts that'll result from this

Reply:
If you want to rename files, it has to be done via locale in a delta folder.
i.e
![image](https://github.com/DeltaV-Station/Delta-v/assets/142105406/757df7e4-0461-41b9-ba4b-53d9b34bb1cd)
![image](https://github.com/DeltaV-Station/Delta-v/assets/142105406/863fc98d-00e7-4d7f-b254-ef5a08076877)